### PR TITLE
Add what CR is in parenthesis.

### PR DESCRIPTION
**Pull request by benjamintanweihao**
I had to look this up to make sure that I didn't misunderstand what CR was.

### CODE REVIEW

**Review by adrian555 — APPROVED** [review]

**Comment by 8bitmp3** [comment]
/assign @adrian555 @shawnzhu 
(for reviewing/LGTM)

gently ping animeshsingh (for approving/LGTM)

**Review by shawnzhu — APPROVED** [review]

**Comment by 8bitmp3** [comment]
@animeshsingh if 👍 please approve, thank you

File: content/en/docs/operator/introduction.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This guide describes the Kubeflow Operator and the current supported releases of
 
 Kubeflow Operator helps deploy, monitor and manage the lifecycle of Kubeflow. Built using the [Operator Framework](https://coreos.com/blog/introducing-operator-framework) which offers an open source toolkit to build, test, package operators and manage the lifecycle of operators.
 
-The operator is currently in incubation phase and is based on this [design doc](https://docs.google.com/document/d/1vNBZOM-gDMpwTbhx0EDU6lDpyUjc7vhT3bdOWWCRjdk/edit#). It is built on top of _KfDef_ CR, and uses _kfctl_ as the nucleus for Controller. Current roadmap for this Operator is listed [here](https://github.com/kubeflow/kfctl/issues/193). The Operator is also [published on OperatorHub](https://operatorhub.io/operator/kubeflow).
+The operator is currently in incubation phase and is based on this [design doc](https://docs.google.com/document/d/1vNBZOM-gDMpwTbhx0EDU6lDpyUjc7vhT3bdOWWCRjdk/edit#). It is built on top of _KfDef_ CR (Custom Resource), and uses _kfctl_ as the nucleus for Controller. Current roadmap for this Operator is listed [here](https://github.com/kubeflow/kfctl/issues/193). The Operator is also [published on OperatorHub](https://operatorhub.io/operator/kubeflow).
```
Comment (joeliedtke) [inline content/en/docs/operator/introduction.md:13]: Thanks for suggesting this change! The convention in tech docs when introducing an abbreviation is to use the full form of the term with the abbreviation in parenthesis. Would you please update this line as follows? 

CR (Custom Resource) -> custom resources (CR)

Also, line 65 also introduces this abbreviation. Would you please make the following change to line 65:

custom resources (CR) -> CR

**Comment by joeliedtke** [comment]
/assign

Preview:
https://deploy-preview-2504--competent-brattain-de2d6d.netlify.app/docs/operator/introduction/

**Pull request closed without merging**